pages.bn/common/ab.md

```diff
@@ -0,0 +1,24 @@
+# ab
+
+> এপাচি এইচটিটিপি সার্ভার বেঞ্চমার্কিং টুল।
+> আরও তথ্য পেতে: <https://httpd.apache.org/docs/current/programs/ab.html>.
+
+- নির্দিষ্ট URL এ ১০০টি HTTP GET অনুরোধ প্রয়ান করুন:
+
+`ab -n {{১০০}} {{url}}`
+
+- URL এ ১০০টি HTTP GET অনুরোধ, ১০টি সময়সার ব্যাচে একে অপরের পরে প্রয়ান করুন:
+
+`ab -n {{১০০}} -c {{১০}} {{url}}`
+
+- URL এ ১০০টি HTTP POST অনুরোধ প্রয়ান করুন, একটি JSON পেলোড ব্যবহার করে:
+
+`ab -n {{১০০}} -T {{application/json}} -p {{পাথ/টু/file.json}} {{url}}`
```

Comment:
```suggestion
`ab -n {{100}} {{url}}`

- URL এ ১০০টি HTTP GET অনুরোধ, ১০টি সময়সার ব্যাচে একে অপরের পরে প্রয়ান করুন:

`ab -n {{100}} -c {{১০}} {{url}}`

- URL এ ১০০টি HTTP POST অনুরোধ প্রয়ান করুন, একটি JSON পেলোড ব্যবহার করে:

`ab -n {{100}} -T {{application/json}} -p {{পাথ/টু/ফাইল.json}} {{url}}`
```

For the placeholder, I would suggest going with the English number rather than the traditional one as there is a high chance that the application might not support this.

Reply:
Ok got it will remember next time.